policy/flask/access_vectors

```diff
@@ -410,6 +410,7 @@ class system
 	syslog_console
 	module_request
 	module_load
+	firmware_load
```

Comment:
Why not add the whole list of newly added permissions?
```
kexec_image_load
kexec_initramfs_load
policy_load
x509_certificate_load
```

Reply:
The pull request is specific about being able to load firmware files, the other permissions can be added later on...